### ISSUE TITLE
Feature: Train in GPU and Use CPU after

### ISSUE THREAD

**Issue by takanovcs** [alankbi/detecto#21]
I trained a model using in GPU and I cant use in CPU to predict, i need to restart all the model using cpu only or force using a pytorch-cpu.

**Comment by alankbi** [comment]
This isn't an officially supported measure, but you could try running the following code: 

```python
model._device = torch.device('cpu')
model._model = model._model.to(torch.device('cpu'))
```
This should set the model to use the CPU from there on out. Let me know if that helps!

**Comment by takanovcs** [comment]
> ```python
> model._device = torch.device('cpu')
> model._model = model._model.to(torch.device('cpu'))
> ```

Thanks but not worked :(

**Comment by takanovcs** [comment]
> This isn't an officially supported measure, but you could try running the following code:
> 
> ```python
> model._device = torch.device('cpu')
> model._model = model._model.to(torch.device('cpu'))
> ```
> 
> This should set the model to use the CPU from there on out. Let me know if that helps!

Thanks, now i tried this.


model = core.Model(labels, device = torch.device('cpu'))  
model._model.load_state_dict(torch.load('./weights/model_weights.pth', map_location=torch.device('cpu')))
